src/microbe_stage/editor/CellStatsIndicator.tscn

```diff
@@ -27,9 +27,11 @@ stretch_mode = 5
 
 [node name="Description" type="Label" parent="."]
 editor_description = "PLACEHOLDER"
+custom_minimum_size = Vector2(130, 0)
 layout_mode = 2
 text = "CELL_STAT_PLACEHOLDER"
 label_settings = ExtResource("2_756k6")
+autowrap_mode = 2
```

Comment:
Is there any particular reason it is not mode 3?

Reply:
The label is so short that I was thinking it would be better to force horizontal scrolling rather than cause a word like "digestion" be split at a random character.

Reply:
Looks good then